### ISSUE TITLE
[1.5.1] Property Hostnames List Flipping (cname_from)

### ISSUE THREAD

**Issue by carnei-ro** [akamai/terraform-provider-akamai#230]
My property has a list of hostnames, at every "terraform plan" the list order of hostnames flips generating a new version of the property.

### Terraform Version
0.14

### Affected Resource(s)
- property

```diff
~ hostnames {
    ~ cname_from             = "foo.bar" -> "baz.bar"
      # (4 unchanged attributes hidden)

      # (1 unchanged block hidden)
  }
~ hostnames {
    ~ cname_from             = "baz.bar" -> "foo.bar"
      # (4 unchanged attributes hidden)

      # (1 unchanged block hidden)
  }
```

**Comment by ericrichtert** [comment]
This construction seems to work:
locals {
  edgehostnames = merge(
    { "T" = {
      (var.hostname) = var.edge_hostname
    } },
    { "A" = {
      (var.hostname) = var.edge_hostname
    } },
    { "P" = {
      (var.hostname) = var.edge_hostname
      "xxxx.nl"                 = var.edge_hostname
      "yyy.nl"                = var.edge_hostname
      "www.xxxx.nl"            = var.edge_hostname
    } }
  )
}

resource "akamai_property" "tf-property-www-wwww-nl" {
  name = "tf-property-${var.functionelehostnaam}"

  product_id  = "prd_Site_Accel"
  contract_id = var.akamai_contract
  group_id    = var.akamai_group

  #hostnames = local.edgehostnames[var.environment]
  dynamic "hostnames" {
    for_each = local.edgehostnames[var.environment]
    content {
      cname_from             = hostnames.key
      cname_to               = hostnames.value
      cert_provisioning_type = "CPS_MANAGED"
    }
  }

  rule_format = "latest"
  rules       = local.rules[var.environment]
}

**Comment by lkowalsk-akamai-com** [comment]
Hi @carnei-ro  
This should be fixed with v1.6.1 release
